pkgs/development/php-packages/gnupg/default.nix

```
@@ -1,10 +1,17 @@
-{ buildPecl, lib, gpgme, file, gnupg, php }:
+{ buildPecl, lib, gpgme, file, gnupg, php, fetchFromGitHub, nix-update-script }:
 
-buildPecl {
+let
+  version = "gnupg-1.5.1";
+in buildPecl {
+  inherit version;
   pname = "gnupg";
 
-  version = "1.5.1";
-  sha256 = "sha256-qZBvRlqyNDyy8xJ+4gnHJ2Ajh0XDSHjZu8FXZIYhklI=";
+  src = fetchFromGitHub {
+    owner = "php-gnupg";
+    repo = "php-gnupg";
+    rev = version;
+    sha256 = "sha256-Bar6no1Ek696E/vfRjlsfafcKNrM61g/ngijIhYXbfg=";
```

```suggestion
    fetchSubmodules = true;
    sha256 = "sha256-kEc0883sYgmAf1mkH0zRjHzUASnZgQvdYE6VzT5X2RI=";
```

We need `fetchSubmodules = true` to get the vendored `phpc` that is present in pecl.net source tarball.